docs/annotations.md

```diff
@@ -0,0 +1,26 @@
+# Annotations
+
+Annotations allow you to modify the default behaviour of the operator.
+
+They can be added to `metadata.annotations` like here:
+
+```
+apiVersion: atlas.mongodb.com/v1
+kind: AtlasProject
+metadata:
+  name: my-project
+  annotations:
+    mongodb.com/atlas-resource-policy: keep
+spec:
+  name: Test Atlas Operator Project
+```
+
+### mongodb.com/atlas-resource-policy=keep
+
+If `mongodb.com/atlas-resource-policy` is set to `keep` operator will not delete the Atlas resource when you delete the k8s resource.
```

Comment:
Maybe "custom resource/k8s custom resource" is better?

Reply:
In that case it's not just any k8s resource but the one which corresponds to Atlas resource.
If there is some confusion here maybe I should change to:
```
when you delete the corresponding k8s resource.
```
Adding "custom" might sound a bit better but I decided not to add it as it actually doesn't mean anything useful, all operator resources are custom.